src/utils/countryZipCode/mergedCountry.js

```diff
@@ -0,0 +1,16 @@
+import COUNTRY_ADDRESS_POSTALS from './index';
+import Country from '../countryCurrency/countriesData.json';
+
+const mergedCountries = () => {
+  const mergedConutryArray = [];
```

Comment:
Fix type `mergedConutryArray` to `mergedCountryArray`.

Reply:
My bad. I have fixed it